### PR TITLE
Find the correct msbuild.exe for VS2017 installs

### PR DESCRIPTION
**Pull request by jvopalensky**
This should fix fully issue #16
(https://github.com/jhaker/nodejs-msbuild/issues/16)

### CODE REVIEW

File: msbuild.js [hunk 1]
```diff
@@ -123,7 +123,42 @@ msbuild.prototype.getMSBuildPath = function(os,processor,version){
...
+	};
+
 	programFilesDir = process.env['programfiles(x86)'] || process.env.PROGRAMFILES;
+
+	// For the msbuild 15.0 version, use the appropriate VS2017 directories
+	if (version === "15.0") {
+
+		// If VSINSTALLDIR env. var cannot be found, see what could be the directory by searching the usual suspects
+		// (while giving higher priority to the VS2017 IDE installs over the Build Tools only install)
+		if (process.env.vsInstallDir === undefined) {
```
Comment (omesrome1) [inline msbuild.js]: I

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "msbuild",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "msbuild wrapper",
   "main": "msbuild.js",
   "scripts": {
```
Comment (omesrome1) [inline package.json]: remove

File: msbuild.js (hunk 1 above)
Comment (jvopalensky) [inline msbuild.js]: Sorry I do not see the difference from my code. Lines 136 to 140 are same as I had them... unless I am missing something.

File: package.json (hunk 2 above)
Comment (jvopalensky) [inline package.json]: Again, I am not sure this is right. I was under the impression that whenever we make small code fixes like this, we need to increase the patch version number. (See https://www.sitepoint.com/semantic-versioning-why-you-should-using/). As far as I see, the current nodejs-msbuild version is 1.0.1 (https://www.npmjs.com/package/msbuild). Thus, this patch should be 1.0.2.

**Pull request merged**